source/baseObject.py

```diff
@@ -72,20 +88,36 @@ def __init__(self,name,bases,dict):
 				else:
 					cache=cacheByDefault
 
+			abstract=dict.get('_abstract_%s'%x,False)
 			if g and not s and not d:

```

Comment:
Would applying deorgan's law make this more clear to a human?
if  g and (not s and not d)
turns into 
if g and not (s or d)
or in other words, if it's a getter, and neither a setter or deleter